src/Clock.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-console */
+import React, { useEffect, useState, useMemo } from 'react';
+
+type Props = {
+  name: string;
+};
+
+export const Clock: React.FC<Props> = ({ name }) => {
+  const [clockState, setClockState] = useState('');
```

Comment:
Clock should have the current time initially
![image](https://user-images.githubusercontent.com/96409563/233022244-0e08d32a-01af-4f38-8c9e-e2b65a667de9.png)